grafana/provisioning/dashboards/qumulo/node.json

```diff
@@ -318,53 +342,39 @@
           },
           "editorMode": "builder",
           "exemplar": false,
-          "expr": "last_over_time(qumulo_node_info{instance=~\"$cluster\"}[24h])",
```

Comment:
Why are these bits removed?

Reply:
This was the node model field, which we didn't add until 6.0.2